src/filesystem/impls/filer/lib/CSSRewriter.js

```diff
@@ -28,6 +28,7 @@ define(function (require, exports, module) {
         function aggregate(content, callback) {
             var urls = [];
             var urlRegex = new RegExp('url\\([\\\'\\"]?([^\\\'\\"\\)]+)[\\\'\\"]?\\)', 'g');
+            var urlQuotesRegex = new RegExp(/[\\\'\\"]([^\\\'\\"\\)]+)[\\\'\\"]/, 'g');
```

Comment:
One thing that this is going to do is double-match every URL in quotes, whether it has `url(` before it or not.  We either need to deal with that here (e.g., tell it to not match on `url(` in this case) or we need to not add duplicates to the `urls` array below.  We could change `urls` to an `Object`, and then use `Object.keys(urls)` to extract an array with no duplicates.